lib/coverband/configuration.rb

```diff
@@ -56,6 +61,12 @@ def s3_secret_access_key
 
     def store
       return @store if @store
+      @store = if ENV['REDIS_URL']
+                 Coverband::Adapters::RedisStore.new(Redis.new(url: ENV['REDIS_URL']))
+               else
+                 Coverband::Adapters::RedisStore.new(Redis.new)
+               end
+      return @store if @store
       raise 'no valid store configured'
     end
```

Comment:
Was thinking it might make sense to try COVERBAND_REDIS_URL then REDIS_URL.

```ruby
def store
  @store ||= Coverband::Adapters::RedisStore.new(Redis.new(url: redis_url))
end

def redis_url
  ENV['COVERBAND_REDIS_URL'] || ENV['REDIS_URL']
end
```

This will give an easy way to specific an alternate redis instance in production.

Reply:
Good call adding this... changed it up a bit to make sure we pull in redis_ttl and redis_namespace as well

Reply:
ok see latest commit I think this is resolved